### DISCUSSION TITLE
[Bug] issues can be closed multiple times without being open

### DISCUSSION THREAD

**Discussion by TobiGr** [community/community#36342] [Projects and Issues]
Open an issue while having no additional rights on the repo.
Close the issue as completed while having admin rights on the repository.
The issue's author is able to close the issue again, and again, and again. The issue does not need to be re-opned to get closed again. See https://github.com/TeamNewPipe/NewPipe/issues/9134#event-7596611141

**Comment by ossanna16** [comment]
Hey there @TobiGr and thank you for reporting this! I'm glad you found our community 🙂
